db/tasks.php

```diff
@@ -0,0 +1,38 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Task definition for tool_mergeusers.
+ * @author    Nicola Vallinoto <n.vallinoto@liguriadigitale.it>
+ * @copyright 2023 Liguria Digitale (https://www.liguriadigitale.it)
+ * @package   tool/mergeusers
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+$tasks = array(
+    array(
+        'classname' => 'tool_mergeusers\task\get_queue_merging_requests',
+        'blocking' => 0,
+        'minute' => '0',
+        'hour' => '0',
```

Comment:
I think this crontab could be improved. You state with this that processing queue to generate adhoc tasks will be only once a day. I think it could happen every hour at least. In fact, I think about it to be run asap, by default. I would rather prefer considering minute `*/5` and hour `*` (so, every 5 minutes, by default).

Reply:
done

Reply:
ok